### ISSUE TITLE
BUG - Breaks Laravel @Page tag in Blade -> DomPDF

### ISSUE THREAD

**Issue by thedoglett** [biati-digital/nova-php-cs-fixer#12]
**Describe the bug**
I'm using DomPDF to generate PDFs on the fly in Laravel. The start of the styling section for this page is as follows:

![Screenshot 2021-03-31 at 09 53 49](https://user-images.githubusercontent.com/40393456/113118173-1fbf7700-9207-11eb-8405-f2c496359e15.jpg)

When I format with PHP-CS-Fixer the tag changes to this:

![Screenshot 2021-03-31 at 09 54 00](https://user-images.githubusercontent.com/40393456/113118208-264dee80-9207-11eb-8ff2-3f63d7de6ca6.jpg)

I wouldn't expect formatting to change @page{ to <blade page|%20%7B>.

Hope that helps.

**Comment by biati-digital** [comment]
Hi, please share the code so I can copy paste it and make some tests.

**Comment by thedoglett** [comment]
Sure. At its simplest you can try formatting this. Make sure its saved as 'filename.blade.php' and then format it.

```
<!DOCTYPE html>
<html>
<head>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<title></title>
	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
	<style>
		@page {
			color: #000;
			font-family: helvetica, arial, verdana sans-serif;
			font-size: 11px;
			font-weight: 300;
			line-height: 1.3;
			margin: 160px 60px 60px 60px;
		}
		</style>
	</head>
</html>
```



This is what happens:



![ezgif com-video-to-gif](https://user-images.githubusercontent.com/40393456/113124875-e6d6d080-920d-11eb-9e25-9bfecb4229d5.gif)



Here's screenshots of my plugin settings:



![Screenshot 2021-03-31 at 10 38 24](https://user-images.githubusercontent.com/40393456/113124281-454f7f00-920d-11eb-9add-77f4dc16f4af.jpg)
![Screenshot 2021-03-31 at 10 39 47](https://user-images.githubusercontent.com/40393456/113124465-716b0000-920d-11eb-992b-3cce9e9a6641.jpg)

**Comment by biati-digital** [comment]
This error should be fixed in the latest version 3.0.2, please update the extension and try it.
